.circleci/config.yml

```diff
@@ -6,14 +6,15 @@ jobs:
       - image: circleci/golang:1.16
     steps:
       - checkout
+      - run:
+          name: Run vet
+          command: go vet ./...
       - run:
           name: Run tests
-          command: |
-            go vet ./...
-            go test ./...
+          command: go test ./...
```

Comment:
Should we put a `when: always` here so we test even if vet fails? Or is it desirable to fail-fast on vet issues?

Reply:
The tests run in about 2 seconds, so I'm not worried either way - I'll add the when